### PR TITLE
Config for frontend vector tiles

### PR DESCRIPTION
**Pull request by ger-benjamin**
Part of GSGMF-1684

Options to create the vectortiles layer. Parameters must match `app.vector_tiles` parameters.

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl [hunk 1]
```diff
@@ -45,6 +45,12 @@ vars:
           coordinatesProjections: *alternate_projections
         gmfPermalinkOptions:
           projectionCodes: *alternate_projections
+        gmfVectorTilesOptions:
+          projection: EPSG:{srid}
+          tileGrid:
+            extent: *extent
+            origin: [2420000, 1350000]
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl:52]: Why is it needed?

Comment (ger-benjamin) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl:52]: To make a correct TileGrid

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl:52]: But it can be extracted from the extent, no?

Comment (ger-benjamin) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl:52]: In Openlayers, we can define extent and origin independently. Origins looks to be the point 0/0/z of the grid, and extent a mask to not display the tiles outside it.
I've kept the original Openlayers TileGrid possibilities. Eventually, I can extract the origins from the extent if the origin is not defined. But I was keeping this poc as low budget as possible.

**Pull request merged**